src/main/docs/guide/aot/aotConfiguration.adoc

```diff
@@ -0,0 +1,35 @@
+
+With https://micronaut-projects.github.io/micronaut-gradle-plugin/latest/#_configuration[Micronaut Gradle Plugin], you can use `aotPlugins` configuration to declare additional AOT modules to be used:
+
+[source,groovy]
+----
+dependencies {
+...
+..
+aotPlugins("io.micronaut.security:micronaut-security-aot")
```

Comment:
I think you will need a version here too, because I think we don't apply the Micronaut BOM to all configurations, but I may be wrong.

Reply:
I added a version and a link to releases https://github.com/micronaut-projects/micronaut-security/pull/868/commits/7bcfa424cb7d2edb50ee3d2d728c30a214e44d96